sample_app/sampleapp_common.c

```diff
@@ -717,13 +717,19 @@ static int app_state_ind (
                switch (err_code)
                {
                case PNET_ERROR_CODE_2_ABORT_AR_CONSUMER_DHT_EXPIRED:
-                  error_description = "AR_CONSUMER_DHT_EXPIRED";
+                  error_description = "ABORT_AR_CONSUMER_DHT_EXPIRED";
                   break;
                case PNET_ERROR_CODE_2_ABORT_AR_CMI_TIMEOUT:
                   error_description = "ABORT_AR_CMI_TIMEOUT";
                   break;
                case PNET_ERROR_CODE_2_ABORT_AR_RELEASE_IND_RECEIVED:
-                  error_description = "Controller sent release request.";
```

Comment:
I prefer human readable descriptive strings for all error cases where we understand what they mean.

Reply:
I have updated all descriptions.